config/sidekiq.yml

```diff
@@ -1,6 +1,8 @@
 :verbose: false
 :concurrency:  1
 :queues:
+  - [rollbacks, 2]
+  - [sync, 2]
   - [default, 5]
```

Comment:
we should add mailers queue and configure the app to use it

Reply:
@matthewford yeah, agree - will do